src/javascript/app/pages/user/account/settings/professional_client.js

```diff
@@ -53,13 +54,13 @@ const professionalClient = (() => {
                 $(popup_selector).find('#btn_accept, #btn_decline').off('click').on('click dblclick', function() {
                     if ($(this).attr('data-value') === 'decline') {
                         $chk_professional.prop('checked', false);
-                        $('#trading_experience_form').setVisibility(1);
+                        $frm_trading_exp.setVisibility(1);
                     }
-                    $('#professional_popup').remove();
+                    $(popup_selector).remove();
```

Comment:
it would be good to also change `const popup_selector    = '#professional_popup';` to `const $popup_selector    = $('#professional_popup');` and use that instead of fetching with jquery every time, especially since there is no other use for the popup_selector id

Reply:
i tried putting it outside, it didn't work :/. it has to be assigned after the element existed, which is after we append the popup to the body